picard/ui/options/matching.py

```diff
@@ -43,6 +44,7 @@ class MatchingOptionsPage(OptionsPage):
         config.FloatOption("setting", "file_lookup_threshold", 0.7),
         config.FloatOption("setting", "cluster_lookup_threshold", 0.7),
         config.FloatOption("setting", "track_matching_threshold", 0.4),
+        config.BoolOption("setting", "show_debug_col_in_match_dialog", False),
```

Comment:
Instead of a generic option, it could be a way to toggle those in the dialog itself, because it's rather hard to find as is.